.github/workflows/vercel.yml

```diff
@@ -34,14 +34,12 @@ jobs:
       - name: Build Project Artifacts
         run: >
           REACT_APP_SENTRY_DSN=${{ secrets.SENTRY_DSN }}
-          REACT_APP_SENTRY_AUTH_TOKEN=${{ secrets.SENTRY_AUTH_TOKEN }}
```

Comment:
I don't think so. I don't see where we use it. But this should be tested.

Reply:
That's still required on Explorer for example.
I had an issue yesterday where the Vercel jobs wouldn't start because the account which created the auth token was deleted, and it does use this variable

https://docs.sentry.io/product/cli/configuration/#configuration-values